src/ember-app.js

```diff
@@ -54,23 +54,30 @@ class EmberApp {
    *
    * @param {string} distPath path to the built Ember app to load
    * @param {Sandbox} [sandboxClass=VMSandbox] sandbox class to use
+   * @param {Object} [sandboxGlobals={}] any additional variables to expose in the sandbox
    */
-  buildSandbox(distPath, sandboxClass) {
+  buildSandbox(distPath, sandboxClass, sandboxGlobals) {
     let Sandbox = sandboxClass || require('./vm-sandbox');
     let sandboxRequire = this.buildWhitelistedRequire(this.moduleWhitelist, distPath);
     let config = this.appConfig;
     function appConfig() {
       return { default: config };
     }
 
-    return new Sandbox({
-      globals: {
-        najax: najax,
-        FastBoot: {
-          require: sandboxRequire,
-          config: appConfig
-        }
+    // add any additional user provided variables in the sandbox
+    let globals = {
+      najax: najax,
+      FastBoot: {
+        require: sandboxRequire,
+        config: appConfig
       }
+    };
+    for (let key in sandboxGlobals) {
+      globals[key] = sandboxGlobals[key];
+    }
```

Comment:
@kristoferbaxter `Object.assign` isn't present in Node 0.12. I wouldn't prefer to add a polyfill for it since Node 0.12 version will dropped soon. We can move to use `Object.assign` once fastboot supports Node 4.x+